src/ArcCommander/ArcConfiguration.fs

```diff
@@ -0,0 +1,119 @@
+﻿namespace ArcCommander
+
+open System.IO
+open Configuration
+
+/// Functionality for providing Configurations to the ArcCommander
+module ArcConfiguration =
```

Comment:
Can you change the functions here to be static members of the `ArcConfiguration` type? That way we dont have ArcConfiguration.ArcConfiguration namespace clutte. you can then also omit the ArcConfiguration from the function names e.g.:

```F#
type ArcConfiguration{...}
	static member create ...
	static member getSection ...
	static member load ...
```

And the use it like this:
```F#
ArcConfiguration.load ..
|> ArcConfiguration.getSection
```